pkg/cmd/stack/translate.go

```diff
@@ -409,15 +411,33 @@ func translateServicePorts(svc *model.Service) []apiv1.ServicePort {
 
 func translateResources(svc *model.Service) apiv1.ResourceRequirements {
 	result := apiv1.ResourceRequirements{}
-	if svc.Resources.CPU.Value.Cmp(resource.MustParse("0")) > 0 {
-		result.Limits = apiv1.ResourceList{}
-		result.Limits[apiv1.ResourceCPU] = svc.Resources.CPU.Value
-	}
-	if svc.Resources.Memory.Value.Cmp(resource.MustParse("0")) > 0 {
-		if result.Limits == nil {
+	if cpuResource, ok := svc.Deploy.Resources.Limits[apiv1.ResourceCPU]; ok {
```

Comment:
here, we should use `svc.Resources.Requests.CPU`, `svc.Resources.Limits.CPU`,...

Reply:
you might need to implement https://github.com/okteto/okteto/issues/1392 first